packages/web-components/fast-components/src/anchor/index.ts

```diff
@@ -55,6 +55,23 @@ export class FASTAnchor extends Anchor {
             this.appearance = "neutral";
         }
     }
+
+    /**
+     * Applies 'icon-only' class when there is only an SVG in the default slot
+     *
+     * @public
+     * @remarks
```

Comment:
you don't need `@remarks` if there is no remark content.

Reply:
```suggestion
```